assignments/arrays.js

```diff
@@ -57,40 +57,56 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // Example for loop:
 // arr = [1,2,3,4];
-// for (i = 0; i < arr.length; i++) {
+// for (let i = 0; i < arr.length; i++) {
 //     arr[i]; // 1,2,3,4
 // }
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make}  ${inventory[32].car_model}`);
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length-1];
+console.log(lastCar);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (let i = 0; i < inventory.length; i++) {
+    carModels.push(inventory[i].car_model); 
+}
+console.log(carModels.sort());
+
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
-let carYears = [];
-console.log();
+
+let carYears= [];
+for (let i = 0; i < inventory.length; i++) {
+    carYears.push(inventory[i].car_year); 
+}
+console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
-let oldCars =[];
-console.log(); 
-
+let oldCars =[]; 
+for (let i = 0; i < carYears.length; i++) {
+    if (carYears[i] < 2000)
+    oldCars.push(carYears[i]); 
```

Comment:
I would strongly recommend always using blocks for if statements.  It prevents you forgetting and adding additional lines afterwards, but because the multi-lines aren't in a block, it won't behave correctly.

The pattern I would recommend is only do the following:
`if (condition) behavor // one line`
```
if (condtion) {
 behavior
} // multiline
```
Some people think you should even include curly brackets on the single line one. (I tend to do that as well). Just my two cents..